netket/vqs/mc/mc_state/expect.py

```diff
@@ -126,13 +125,19 @@ def logpsi(w, σ):
     def log_pdf(w, σ):
         return machine_pow * model_apply_fun({"params": w, **model_state}, σ).real
 
-    _, Ō_stats = nkjax.expect(
-        log_pdf,
-        partial(local_value_kernel, logpsi),
-        parameters,
-        σ,
-        local_value_args,
-        n_chains=σ_shape[0],
-    )
+    # TODO: Broken until google/jax#11916 is resolved.
+    # should uncomment and remove code below once this is fixed
+    # _, Ō_stats = nkjax.expect(
+    #    log_pdf,
+    #    partial(local_value_kernel, logpsi),
+    #    parameters,
+    #    σ,
+    #    local_value_args,
+    #    n_chains=σ_shape[0],
+    # )
+
+    L_σ = local_value_kernel(logpsi, parameters, σ, local_value_args)
```

Comment:
I actually find this clearer than the previous version, can we leave this? :) or is there a performance hit?

Reply:
They are equivalent now (minus jax lost features) as long as you don't compute gradients of this code (which we never do, as we rewrite it in expect and grad).

If you compute the gradient you need to use expect otherwise you get the wrong gradient.

Originally I wanted to avoid writing this code twice but this never happened so we can keep this.